samples/net/zperf/prj_netusb.conf

```diff
@@ -6,8 +6,8 @@ CONFIG_NET_UDP=y
 CONFIG_NET_TCP=y
 CONFIG_NET_STATISTICS=y
 
-CONFIG_NET_PKT_RX_COUNT=100
-CONFIG_NET_PKT_TX_COUNT=100
```

Comment:
These changes are not related to TCP issue, but sanitycheck fails to build because of extra 272Bytes in RAM.